### PR TITLE
flake8: Fix redefinition of unused variables

### PR DESCRIPTION
**Pull request by cclauss**
Fix flake8 error F811 -- https://flake8.pycqa.org/en/latest/user/error-codes.html

### CODE REVIEW

File: openlibrary/plugins/upstream/jsdef.py [hunk 1]
```diff
@@ -63,9 +63,11 @@
     SuiteNode,
     DefNode,
     PythonTokenizer,
-    INDENT,
+    # INDENT,
 )
 
+INDENT = "    "
```
Comment (mekarpeles) [inline openlibrary/plugins/upstream/jsdef.py:69]: Is this export needed upstream?

**Comment by mekarpeles** [comment]
lgtm other than 1 Q about `INDENT` movement

File: openlibrary/plugins/upstream/jsdef.py (hunk 1 above)
Comment (cclauss) [reply openlibrary/plugins/upstream/jsdef.py:69]: Across the whole repo...
% `git grep INDENT`
```
openlibrary/plugins/upstream/jsdef.py:    INDENT,
openlibrary/plugins/upstream/jsdef.py:INDENT = "    "
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
openlibrary/plugins/upstream/jsdef.py:        text += self.jsemit(node.suite, indent + INDENT)
```

**Review by cdrini — APPROVED** [review]
lgtm; nothing looks risky. Not testing

**Comment by cdrini** [comment]
Ooops I didn't see you'd reviewed it Mek!

**Pull request merged**